in_toto/models/layout.py

```diff
@@ -38,7 +38,7 @@
 from dateutil.parser import parse
 
 import in_toto.matchrule_validators
-import securesystemslib.exceptions
+from securesystemslib.exceptions import FormatError
```

Comment:
When I use "import securesystemslib.exceptions.FormatError" instead of "from securesystemslib.exceptions import FormatError",
it gives me "ImportError: No module named FormatError"
Any ideas?